packages/rx_bloc_cli/bin/compile.sh

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env sh
-pub run mason bundle -t dart mason_templates/bricks/rx_bloc_base
+flutter pub run mason bundle -t dart mason_templates/bricks/rx_bloc_base
 cp rx_bloc_base_bundle.dart lib/src/templates/
 rm rx_bloc_base_bundle.dart
-pub global activate -s path . --overwrite
-rx_bloc_cli create --org com.primeholding --project-name test_app --include-analytics true example/test_app
+flutter pub global activate -s path . --overwrite
+flutter pub global run rx_bloc_cli create --org com.primeholding --project-name test_app --include-analytics true example/test_app
```

Comment:
I've had issues getting this to work on Windows, since it's a bash script and global executables on Windows are stored as .bat, but it's been reverted